### PR TITLE
`LiSSA`: First version that puts sketch recognition into a Pipeline in ArDoCo

### PR DESCRIPTION
**Pull request by dfuchss**
This PR is the starting point for a new Pipeline within the ArDoCo Framework. It is the migration of the [LiSSA-Approach](https://github.com/lissa-approach/lissa-core) to the ArDoCo Framework.
This PR adds a Diagram Recognition stage that uses object detection and OCR to analyze sketches and diagrams.

### CODE REVIEW

File: docs/Home.md [hunk 1]
```diff
@@ -1,23 +1,49 @@
...
+The `complete` profile includes all the requirements that the special profiles also need. This profile is activated by
+default.
+
+All profiles require JDK 17.
+
+The dependencies of the other profiles at a glance:
+
+* tlr: -
+* inconsistency: -
+* lissa (LInking Sketches and Software Architecture): docker (local
```
Comment (Hossiphi) [inline docs/Home.md:23]:
```suggestion
* lissa (LInking Sketches and Software Architecture): Docker (local
```

File: docs/LiSSA.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+The LiSSA approach aims to connect sketches and informal diagrams (such as class diagrams, component diagrams, ..) with
```
Comment (Hossiphi) [inline docs/LiSSA.md:1]:
```suggestion
The LiSSA approach aims to connect sketches and informal diagrams (such as class diagrams, component diagrams, ...) with
```

**Review by Hossiphi — APPROVED** [review]

File: docs/Home.md [hunk 3]
```diff
@@ -1,23 +1,49 @@
...
+a component or a relation. To do so, ArDoCo first creates trace links and then makes use of them and other information
+to identify inconsistencies.
 
 You can find [ArDoCo on GitHub](https://github.com/ArDoCo).
 
 Before contributing, please read the [Quickstart Guide](quickstart).
 
 JavaDocs can be found [here](https://ardoco.github.io/Core-Docs/).
 
+## Requirements
```
Comment (dfuchss) [inline docs/Home.md:12]:
```suggestion
## System Requirements
```

File: stages/diagram-recognition/src/main/kotlin/edu/kit/kastel/mcse/ardoco/lissa/diagramrecognition/informants/OCRInformant.kt [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+import org.apache.hc.client5.http.classic.methods.HttpPost
+import org.apache.hc.client5.http.entity.mime.MultipartEntityBuilder
+import org.apache.hc.core5.http.ContentType
+import org.apache.hc.core5.http.HttpEntity
+import org.apache.hc.core5.net.URIBuilder
+import java.io.ByteArrayInputStream
+import java.io.InputStream
+import java.net.URI
+
+class OCRInformant(dataRepository: DataRepository) : ImageProcessingDockerInformant(
```
Comment (Gram21) [inline stages/diagram-recognition/src/main/kotlin/edu/kit/kastel/mcse/ardoco/lissa/diagramrecognition/informants/OCRInformant.kt:19]: IMHO, "correct" capitalization is "OcrInformant"

**Pull request merged**